src/lib/modernExtend.ts

```diff
@@ -779,6 +787,48 @@ export function commandsLevelCtrl(args?: CommandsLevelCtrl): ModernExtend {
         fz.command_stop,
     ];
 
+    if (args.legacyClick) {
```

Comment:
```suggestion
    if (args.legacyAction) {
```

Isn't this a better name?